axltempl/files/drupal/settings.env.php

```diff
@@ -1,5 +1,6 @@
 <?php
 
+// @codingStandardsIgnoreFile
```

Comment:
```suggestion
/**
 * @file
 * Load settings from environment variables.
 * @see /.env.example
 */
```

How about this instead?